### DISCUSSION TITLE
How to import tailwind predefined color palette?

### DISCUSSION THREAD

**Discussion by ArtemAnufiev** [tailpress/tailpress#202] [Q&A]
Ho wo extend your tailwind.config.js and add these values? Many thanks for your project?

const colors = require('tailwindcss/colors')

		colors: {
			primary: '#6637EE',
			secondary: '#FD5F0E',
			white: colors.white,
			indigo: colors.indigo,
			gray: colors.gray
		}

**Comment by jeffreyvr** [comment]
I add my theme colors in the `theme.json` file. Then they are automatically added to your Tailwind colors through this part: https://github.com/jeffreyvr/tailpress/blob/master/tailwind.config.js#L22
